header-only/layout-builder/awkward/LayoutBuilder.h

```diff
@@ -19,7 +19,7 @@ namespace awkward {
 
     /// @brief Object of {@link BuilderOptions BuilderOptions} which sets the
     /// values of the default options.
-    awkward::BuilderOptions default_options(1024, 1);
+    inline awkward::BuilderOptions default_options(1024, 1);
```

Comment:
This is an inlined variable - does it need to be?

Reply:
When we include `LayoutBuilder.h` in multiple source files, it results in multiple definitions of the `default_options` leading to linker errors. So yes it will need to be inline.